respy/state_space.py

```diff
@@ -90,6 +92,16 @@ def _create_is_inadmissible(self, optim_paras, options):
 
         is_inadmissible = core[optim_paras["choices"]].to_numpy(dtype=np.bool)
 
+        if np.any(is_inadmissible) and optim_paras["inadmissibility_penalty"] is None:
+            warnings.warn(
```

Comment:
Just out of curiosity, this is probably not going to work anymore once we have state-specific choice sets, right? (@mo2561057 , just to keep you in the loop) I think the approximate solution is one of the upcoming bottlenecks for us, so I write Prof. Garcke and requested some help in the form of a Master thesis from the INS.

Reply:
If we have the state-specific choice set, I hope the `is_inadmissible` matrices are gone. Instead, the penalty would be solely applied in the interpolation. Thus, we would check whether interpolation is requested and whether there are different choice sets.